stubs/Driver/CursorInterface.stub.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace MongoDB\Driver;
+
+use Iterator;
+
+/**
+ * @template TValue of array|object
+ * @template-implements Iterator<int, TValue>
+ */
+interface CursorInterface extends Iterator
+{
+    /**
+     * @return TValue|null
+     * @psalm-ignore-nullable-return
+     */
+    public function current(): array|object|null;
+
+    public function getId(): \MongoDB\BSON\Int64;
```

Comment:
I imagine this was copied from PHPC which doesn't support `use` statements in its stubs, but would Psalm normally prefer a `use` statement for this?

Reply:
Yep, good point. Will update this